pyproject.toml

```
@@ -5,7 +5,7 @@ build-backend = "flit_core.buildapi"
 [project]
 # Check https://flit.readthedocs.io/en/latest/pyproject_toml.html for all available sections
 name = "ansys-mechanical-core"
-version = "0.9.dev1"
+version = "0.10.0"
```

```suggestion
version = "0.11.dev0"
```

The main branch is usually one minor version ahead from the latest released version